app/scripts-browserify/auth.js

```diff
@@ -17,10 +17,19 @@ $(function() {
   }, lock(), verify());
 
   function lock() {
+    let languageDictionary = {
+      title: config.siteTitle
+    };
+
+    if (config.auth0.mustAcceptTermsPath) {
+      languageDictionary.signUpTerms = `Jeg accepterer <a href="${config.auth0.mustAcceptTermsPath}" target="_new">Regler og vilkår</a> for deltagelse på kbhbilleder.dk`;
```

Comment:
Any reason why this does not use `config.siteTitle` or the like instead of using the site url directly? This might make it usable upstream as wel..